src/planscape/boundary/models.py

```diff
@@ -0,0 +1,26 @@
+from django.contrib.gis.db import models
+
+
+class Boundary(models.Model):
+    """A Boundary object is a collection of polygons with a name."""
```

Comment:
should we use a numerical id as an internal key?

Reply:
I agree.  Mind if I follow this up with a different PR, since it changes a lot?

Reply:
sg!